elasticsearch/src/main/java/com/gentics/mesh/search/index/node/NodeIndexHandler.java

```diff
@@ -393,11 +407,78 @@ private void store(Set<Single<String>> obs, Node node, String releaseUuid, Conta
 			} else {
 				obs.add(storeContainer(container, releaseUuid, type));
 			}
-			// obs.add(sanitizeIndex(node, container, context.getLanguageTag()).toCompletable());
 		} else {
 			for (NodeGraphFieldContainer container : node.getGraphFieldContainersIt(releaseUuid, type)) {
 				obs.add(storeContainer(container, releaseUuid, type));
-				// obs.add(sanitizeIndex(node, container, context.getLanguageTag()).toCompletable());
+			}
+		}
+
+	}
+
+	/**
+	 * Step 1 - Check whether we need to handle all releases.
+	 * 
+	 * @param obs
+	 * @param node
+	 * @param context
+	 */
+	private void storeForBulk(Set<Single<IndexBulkEntry>> obs, Node node, GenericEntryContext context) {
+		if (context.getReleaseUuid() == null) {
+			for (Release release : node.getProject().getReleaseRoot().findAllIt()) {
+				storeForBulk(obs, node, release.getUuid(), context);
+			}
+		} else {
+			storeForBulk(obs, node, context.getReleaseUuid(), context);
+		}
+	}
+
```

Comment:
Return the set instead of adding it to a parameter. Remove the obs parameter

Reply:
Same for all other methods below

Reply:
done